v2/concordium/protocol-level-tokens.proto

```diff
@@ -159,14 +159,12 @@ message CreatePLT {
   TokenId token_id = 1;
   // The hash that identifies the token module implementation.
   TokenModuleRef token_module = 2;
-  // The address of the account that will govern the token.
-  AccountAddress governance_account = 3;
```

Comment:
Also need to remove `issuer` from `TokenState` right?

Reply:
yes, correct.